scripts/spack/configs/linux_ubuntu_20/packages.yaml

```diff
@@ -79,8 +80,8 @@ packages:
   cuda:
     buildable: false
     externals:
-    - spec: cuda@11.2.0
-      prefix: /usr/local/cuda-11.2
+    - spec: cuda@11.7.0
```

Comment:
@jamiebramwell do you have 11.7 installed on your machine? Was this moved forward for a reason (bug in 11.2, etc)?

Reply:
I don't have cuda installed on my machine at all. I believe this was for @samuelpmishLLNL 's local build.

Reply:
k.. at some point in the future we may have to make these user-specific.

Reply:
I wish these tools would just identify the version number automatically when you point it at a directory, it's just a one-liner to deduce this information:

```sh
sam@provolone:~$ /usr/local/cuda/bin/nvcc --version | grep -o -P "release.{0,5}" | grep -o "....$"
11.7
```

instead we're requiring users to either:
- have incredibly specific versions of certain libraries
- manually dive into these config files to build their dependencies

both of which are unpleasant

Reply:
that being said, I didn't intend to include this file in the PR, I can revert it back to 11.2 (although nvcc 11.7 does include a number of fixes for compiler bugs we submitted to NVIDIA)

Reply:
For now, I think this could either be a new config directory (`linux_ubuntu_20_cuda`?) or just removed from this PR.